### ISSUE TITLE
does not include dots in the middle of a path

### ISSUE THREAD

**Issue by KTibow** [maxwroc/vanilla-pattern-lock#8]
this
![image](https://github.com/maxwroc/vanilla-pattern-lock/assets/10727862/ef6314cc-6ed8-41bb-b68c-af38d5ea5c26)
should be auto-converted to this
![image](https://github.com/maxwroc/vanilla-pattern-lock/assets/10727862/d5a4286a-1f38-4a4c-bee8-aa7100fd89a5)
that's what android does

**Comment by maxwroc** [comment]
How did you achieve such result? (when the middle node is not selected)

Have you intentionally moved your finger /mouse around it?

If so then this is by design. It could be added as the additional option/setting though.

**Comment by KTibow** [comment]
yeah i had to intentionally move my finger around it - i was just bringing this up because GH showed me this repo, and i wanted to see how close it was to real android pattern unlock (which includes the dots you go around)
